### ISSUE TITLE
Jio tv most popular channels are not working 

### ISSUE THREAD

**Issue by syedirshad20** [botallen/repository.botallen#323]
**Your Environment**
- Add-on name: [e.g. Jiotv]
- Add-on version: [e.g. 2.0.0]
- Kodi version: [e.g. Kodi 19]
- Operating system version/name: [e.g. Windows 10, LibreElec 9.0, etc...]
- Using from country: [eg. India, USA, etc...]

**Describe the bug**

**To Reproduce**
4. See an error

**Expected behavior**

**Screenshots**

**Additional context**

**Debug log**

The debug log can be found from this link:

**Comment by rashnk** [comment]
Yes, they moved sports channel to jio cinema, that may be the reason

**Comment by syedirshad20** [comment]
Colors , zee channels etc. Not working

**Comment by syedirshad20** [comment]
>

**Comment by mhunny** [comment]
Try disabling Extra HD Channels under under Jio add on settings and then re-add your fav channels and try.
